pkg/config/linters_config.go

```diff
@@ -172,3 +175,11 @@ type RequiredFieldsConfig struct {
 	// When otherwise not specified, the default value is "SuggestFix".
 	PointerPolicy RequiredFieldPointerPolicy `json:"pointerPolicy"`
 }
+
+// StatusOptionalConfig contains configuration for the statusoptional linter.
+type StatusOptionalConfig struct {
+	// preferredOptionalMarker is the preferred marker to use for optional fields.
+	// If this field is not set, the default value is "optional".
+	// Valid values are "optional" and "kubebuilder:validation:Optional".
```

Comment:
And the K8s version now right? Or do you not want to include that in the documentation until the declarative validation project is further along?